csrc/quantization/vectorization_utils.cuh

```diff
@@ -0,0 +1,51 @@
+// vector_utils.cuh
+#pragma once
+#include "vectorization.cuh"
+
+namespace vllm {
+
+template <typename InT, typename OutT, typename VecOp, typename ScaOp>
+__device__ inline void vectorize_with_alignment(
+    const InT* in, OutT* out, int len, int tid, int stride,
+    VecOp&& vec_op,    // vec_n_t<InT,16> -> vec_n_t<OutT,16>
+    ScaOp&& sca_op) {  // InT -> OutT
```

Comment:
I really don't `scalar_op` is too long 😅

Reply:
Got it, fixed!